src/routes/Routes.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+import page from 'page';
+import Index from './index.svelte';
+import Explore from './explore.svelte';
+
+let component;
+page('/', (p) => {
+  if (p.hash === 'explore') {
+    component = Explore;
+  } else {
+    component = Index;
+  }
+});
```

Comment:
Would this approach scale? I came around to liking this pattern best, which we use in GLAM:

https://spiffy.tech/blog/a-minimal-svelte-router/

Reply:
For context, I had transfered over my own project template from my personal account. But given all this, I am not sure I would recommend to anyone the approach we took w/ GLAM at this point for a pure client-side app. It clearly requires more ops intervention which can be costly and full of friction. I'm also not convinced it scales well, but this is a starter template, so we can remove this to just prevent any confusion. ([This post](https://reactarmory.com/answers/push-state-vs-hash-based-routing) has informed some of my thinking since GLAM.)

Reply:
Sure, was just experimenting with only paying attention to the hashbang, but it looks like in the example you linked to, `page.js` has a `{hashbang: true}` as well. I would opt for that in this template since I think this is going to be primarily useful (and most-used) for pure client-side routing, and full path routing is a non-starter in the majority of cases.

Reply:
Yeah, I should have said that I didn't use `{ hashbang: true }` in GLAM, but we could use it here if you like. I'll let you decide what to do, if anything, with these lines.